From: Arturo Borrero Gonzalez <arturo.borrero.glez@gmail.com>
Subject: Debian default configuration
 This patch sets Debian defaults for liquidprompt configuration.
Forwarded: not-needed
Last-Update: 2015-06-24

--- a/liquidpromptrc-dist
+++ b/liquidpromptrc-dist
@@ -5,7 +5,8 @@
 
 # If you want to use different themes and features,
 # you can load the corresponding files here:
-#source ~/.config/liquidprompt/nojhan.theme
+# Debian default loads debian.theme
+source /usr/share/liquidprompt/debian.theme
 #LP_PS1_FILE="~/.config/liquidprompt/nojhan.ps1"
 
 #############
@@ -35,7 +36,8 @@
 # Do you want to display the hostname, even if not connected through network?
 # Defaults to 0 (do not display hostname when locally connected)
 # set to 1 if you want to always see the hostname
-LP_HOSTNAME_ALWAYS=0
+# Debian default is 1
+LP_HOSTNAME_ALWAYS=1
 
 # Do you want to display the user, even if he is the same than the logged one?
 # Defaults to 1 (always display the user)
@@ -101,7 +103,8 @@
 
 # Show runtime of the last command if over LP_RUNTIME_THRESHOLD
 # Recommended value is 0
-LP_ENABLE_RUNTIME=0
+# Debian default is 1, only active in bash
+LP_ENABLE_RUNTIME=1
 
 # Minimal runtime to be displayed
 # Recommended value is 2
--- a/liquidprompt
+++ b/liquidprompt
@@ -415,7 +415,6 @@
 $_LP_ENABLE_SCREEN || $_LP_ENABLE_TMUX ; _lp_bool _LP_ENABLE_DETACHED_SESSIONS $?
 
 if [[ "$LP_ENABLE_RUNTIME" == 1 ]] && ! $_LP_SHELL_bash; then
-    echo Unfortunately, runtime printing for zsh is not yet supported. Turn LP_ENABLE_RUNTIME off in your config to hide this message.
     LP_ENABLE_RUNTIME=0
 fi
 
