From: Arturo Borrero Gonzalez <arturo@debian.org>
Subject: Debian default configuration
 This patch sets Debian defaults for liquidprompt configuration.
Forwarded: not-needed
Last-Update: 2015-06-24

--- a/liquidpromptrc-dist
+++ b/liquidpromptrc-dist
@@ -5,7 +5,8 @@
 
 # If you want to use different themes and features,
 # you can load the corresponding files here:
-#source ~/.config/liquidprompt/nojhan.theme
+# Debian default loads debian.theme
+source /usr/share/liquidprompt/debian.theme
 #LP_PS1_FILE=~/.config/liquidprompt/nojhan.ps1
 
 #############
@@ -36,7 +37,8 @@
 # Defaults to 0 (do not display hostname when locally connected)
 # set to 1 if you want to always see the hostname
 # set to -1 if you want to never see the hostname
-LP_HOSTNAME_ALWAYS=0
+# Debian default is 1
+LP_HOSTNAME_ALWAYS=1
 
 # Use the FQDN instead of the short hostname if the hostname is displayed
 LP_ENABLE_FQDN=0
@@ -110,7 +112,8 @@
 
 # Show runtime of the last command if over LP_RUNTIME_THRESHOLD
 # Recommended value is 0
-LP_ENABLE_RUNTIME=0
+# Debian default is 1, only active in bash
+LP_ENABLE_RUNTIME=1
 
 # Minimal runtime to be displayed
 # Recommended value is 2
